client/core/core.go

```diff
@@ -2070,14 +2070,14 @@ func (c *Core) Register(form *RegisterForm) (*RegisterResult, error) {
 	requiredConfs := dc.cfg.RegFeeConfirms
 	dc.cfgMtx.RUnlock()
 
+	details := fmt.Sprintf("Waiting for %d confirmations before trading at %s", requiredConfs, dc.acct.host)
+	c.notify(newFeePaymentNote(SubjectFeePaymentInProgress, details, db.Success, dc.acct.host))
+
```

Comment:
Since `verifyRegistrationFee` is async, having this `newFeePaymentNote` after it creates a race with the notification send by `verifyRegistrationFee`.

Reply:
I can't recall, but this seems fine.